crates/relayer/src/client_state.rs

```diff
@@ -100,9 +100,9 @@ impl AnyClientState {
     pub fn trust_threshold(&self) -> Option<TrustThreshold> {
         match self {
             AnyClientState::Tendermint(state) => Some(state.trust_threshold),
-            AnyClientState::Eth(_) => None,
-            AnyClientState::Ckb(_) => None,
-            AnyClientState::Axon(_) => TrustThreshold::new(1, 2).ok(),
+            AnyClientState::Eth(_) => Some(TrustThreshold::ONE_THIRD),
+            AnyClientState::Ckb(_) => Some(TrustThreshold::ONE_THIRD),
```

Comment:
https://github.com/synapseweb3/forcerelay/blob/main/crates/relayer-types/src/core/ics02_client/trust_threshold.rs#L29-L32

Why is the TrustThreshold of `AnyClientState::Ckb` defined as ONE_THIRD?
Is there any research on this topic or is it just an TODO item?

Reply:
can be recognized as just a TODO item to bypass some process in Hermes framework, I'll leave a comment to clarify this 
specific use case